### ISSUE TITLE
Nvim-tree does not quit if you use FZF to find file

### ISSUE THREAD

**Issue by hqkhan** [kyazdani42/nvim-tree.lua#626]
I have an issue where if I have open `nvim` and navigate to a file using `fzf`, the tree stays open and moves to the right side of the screen. Relevant settings:

```
let g:nvim_tree_quit_on_open = 1 "0 by default, closes the tree when you open a file
let g:nvim_tree_auto_open = 1 "0 by default, opens the tree when typing `vim $DIR` or `vim`
```
`nvim-tree` does close itself if you navigate to a file using the tree itself but I was hoping that it close the tree no matter how I got to a file. Do I have to issue something custom using `au BufReadPost` maybe?

**Comment by neurosnap** [comment]
Came to the issues to look for this issue.  An earlier version did not have this problem, this only came up recently when I updated nvim-tree.

**Comment by kyazdani42** [comment]
fixed in `1.5.2` (29e5b754b7250aded60)
